src/components/Heading/Heading.scss

```diff
@@ -1,5 +1,33 @@
+@import 'foundation/colors';
+
+$color-custom-gray-normal: theme-color(m-grey, 900);
+$color-custom-gray-darker: theme-color(m-grey, 700);
+$color-custom-gray-darkest: theme-color(m-grey, 800);
+$color-custom-gray-mid: theme-color(m-grey, 500);
+$color-custom-gray-reverse: #fff;
+$color-custom-gray-danger: theme-color(m-red, 800);
```

Comment:
@Farhan-emgage these won't work here. Details on proper var naming and Delicious theme color usage provided in previous comments.